### PR TITLE
feature: push version

### PR DESCRIPTION
**Pull request by JYisus**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [ ] I have created tests for my code changes, and the tests are passing.
- [ ] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by kafkaphoenix — DISMISSED** [review]

File: api/api.go [hunk 1]
```diff
@@ -5,25 +5,32 @@ import (
...
+	versionClient   kai.VersionClient
 }
 
 // ProcessRegistry access to methods to interact with the Process Registry.
 func (a *KAI) ProcessRegistry() processregistry.APIClient {
 	return a.processRegistry
 }
 
-// ProductClient access to methods to interact with the Process Registry.
+// ProductClient access to methods to interact with the product's API.
```
Comment (alexrodfe) [inline api/api.go:24]: +1 :+1:

File: cmd/product/version/version.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package version
+
+import (
+	"github.com/MakeNowJust/heredoc"
+	"github.com/konstellation-io/kli/internal/logging"
+	"github.com/spf13/cobra"
+)
+
+// NewVersionCmd creates a new command to handle 'product' subcommands.
```
Comment (alexrodfe) [inline cmd/product/version/version.go:9]: perhaps we could add to "handle 'product' subcommands related to version' ?

**Review by alexrodfe — DISMISSED** [review]
Great!
Just a suggestion ahead. The code is really clean :metal:

File: cmd/product/version/push.go [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+)
+
+const (
+	_versionFlag     = "version"
+	_descriptionFlag = "description"
+	_initLocalFlag   = "init-local"
+	_localPathFlag   = "local-path"
+)
+
+// NewPushCmd creates a new command to create a new product.
```
Comment (alexrodfe) [inline cmd/product/version/push.go:18]: a new product version*

Comment (JYisus) [reply cmd/product/version/push.go:18]: TBH i'm not a huge fan of these comments when we are not developing a public API. I'll correct them in the cmd package, but I think we could save time if we stop adding them in the rest of services.

**Review by alexrodfe — APPROVED** [review]

**Pull request merged**